GetStream/stream-chat-android
# [3220] Fix see more click handler

### 🎯 Goal

Don't consume click events when clicking on the see more tile of the gallery collage.

### 🎨 UI Changes

_Add relevant screenshots_

| Before | After |
| --- | --- |
| ![photo_2022-03-17 17 45 18](https://user-images.githubusercontent.com/9600921/158830318-7a0e8bdf-1c4a-4a0d-a099-200ffb4579ca.jpeg) | ![photo_2022-03-17 17 45 21](https://user-images.githubusercontent.com/9600921/158830314-29114833-d99f-437e-bccd-03722491265e.jpeg) |

### 🧪 Testing

1. Navigate to the message list
2. Locate a message with 4+ attachments
3. Click on the "see more" tile (e.g. "2+")
4. At this point you should be taken to the image preview screen

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
